lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -124,6 +124,7 @@ private function _add_external_html_elements()
 
         $prefix = $this->_config->get('tinymce_url');
         midcom::get()->head->add_jsfile("{$prefix}/tinymce.min.js", true);
+        midcom::get()->head->add_jsfile("{$prefix}/imagetools_functions.js", true);
```

Comment:
please move the file outside of the tinymce folder, otherwise it would get lost the next time we update tinymce